studybuddies/src/app/services/class.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import { Room } from "../models/class";
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202914)
```suggestion
  providedIn: "root"
```